doc/dependency_decisions.yml

```diff
@@ -630,4 +630,9 @@
 - - :approve
   - WHY=0BSD
   - *16
-
+- - :remove_package
+  - canvas
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2022-10-28 11:26:31.331728000 Z
```

Comment:
why do we need a decision to remove package canvas?

Reply:
If I understood correctly, this file keeps track of what packages are added and removed. Since we don't use canvas anymore I thought it was best to remove it. There is an entry approving it:
https://github.com/josemigallas/porta/blob/61d162aeb78f7768d6f1f783e3610404149be533/doc/dependency_decisions.yml#L581-L587

Reply:
I think we need to add/remove there individual packages only when their licenses are not approved by default. Strange that some packages were individually approved. Honestly I'd rather remove them from file than making it longer. We have git history anyway.

Reply:
Unfortunately there is not much information about it https://github.com/pivotal/LicenseFinder#decisions-file. It's supposed to be handled by `license_finder` alone 🤷🏻‍♂️ but I don't really understand the importance of it